app/controllers/price_groups_controller.rb

```diff
@@ -108,12 +105,30 @@ def destroy
 
   private
 
+  def price_group_members(search_term = nil)
+    unless SettingsHelper.feature_on?(:user_based_price_groups) && @price_group_ability.can?(:read, UserPriceGroupMember)
+      raise ActiveRecord::RecordNotFound
+    end
+
+    @user_members = @price_group.user_price_group_members
+                                .includes(:user)
+                                .joins(:user)
+
+    if search_term.present?
+      @user_members = @user_members.where("LOWER(users.last_name) LIKE :search OR LOWER(users.first_name) LIKE :search OR LOWER(users.username) LIKE :search", search: search_term)
+    end
+
+    @user_members = @user_members.merge(User.sort_last_first)
+    @user_members = paginate(@user_members)
+    @tab = :users
```

Comment:
I would leave this part out of this method, even though both actions do it, it's not really part of finding user members.